### DISCUSSION TITLE
Enabling before / after steps in report

### DISCUSSION THREAD

**Discussion by sethvanwykJET** [allure-framework/allure2#2055] [Questions & Support]
## Summary

Good day.

I'm using codeceptJS and the runner has a before / after method that can be used. It looks like this isn't considered as "part of the test" in all circumstances. Is there a config I can enable to have these show up in the allure report?

**Comment by baev** [comment]
As far as I know, Allure CodeceptJS integration doesn't support test fixtures yet.

**Reply by sethvanwykJET** [reply]
Hmm.

In the past with an older version before migrating to Allure 2, it was able to show the before / after steps in the reporting. But it doesn't seem to be present anymore.
